src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/ElementsSection.tsx

```diff
@@ -122,6 +122,8 @@ const ElementsSection = (props: {
   };
 
   const handleAddDataElement = (sourceCriteria) => {
+    delete sourceCriteria._id;
```

Comment:
rather than manipulating the input argument, do it work if we set `_id` to `null` on the next line? Or `delete` from `data`?